features/configuration_files/exclude_paths_directives.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Exclude paths directives
+  In order to avoid Reek wasting time on files that cannot be fixed
+  As a user
+  I want to be able to exclude specific paths from being checked
+
+  Scenario: Exclude some paths
+    Given a file named "config.reek" with:
+      """
+      ---
+      exclude_paths:
+        - bad_files_live_here
+      """
+    And a file named "bad_files_live_here/smelly.rb" with:
+      """
+      class Smelly
+        # Should report IrresponsibleModule
+        def foo(arg); end # Should report UnusedParameter
+        def x; end        # Should not report UncommunicativeMethodName
```

Comment:
I just copied another smelly file example as a black box. I'll make it smaller since we only need the one smell.